internal/dockerauth/api.go

```diff
@@ -129,7 +130,7 @@ func (h *handler) Token(p httprequest.Params, req *tokenRequest) (*tokenResponse
 			if err == nil {
 				filteredActions = append(filteredActions, a)
 			} else {
-				logger.Infof("check failed: %v", err)
+				logger.Infof("docker token check failed for operation %q: %v", a, err)
```

Comment:
I'm not certain this should be logged at info level either.

Reply:
changed to Debug.